filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -163,9 +163,29 @@ VulkanDriver::VulkanDriver(VulkanPlatform* platform,
 
     // Choose a depth format that meets our requirements. Take care not to include stencil formats
     // just yet, since that would require a corollary change to the "aspect" flags for the VkImage.
-    mContext.depthFormat = findSupportedFormat(mContext,
+    mContext.finalDepthFormat = findSupportedFormat(mContext,
         { VK_FORMAT_D32_SFLOAT, VK_FORMAT_X8_D24_UNORM_PACK32 },
         VK_IMAGE_TILING_OPTIMAL, VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT);
+
+    // For diagnostic purposes, print useful information about available depth formats.
+    // Note that Vulkan is more constrained than OpenGL ES 3.1 in this area.
+    #if VK_ENABLE_VALIDATION
```

Comment:
Left-align preprocessor directives? Same for VulkanHandles.cpp

Reply:
Oh right.  Done.